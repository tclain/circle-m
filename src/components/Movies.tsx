import { Flex } from "@chakra-ui/core";
import React from "react";
import { Movie as MovieType } from "../types";
import { Movie } from "./Movie";

export interface MoviesProps {
  movies: MovieType[];
}

export const Movies: React.FC<MoviesProps> = ({ movies }) => {
  return (
    <Flex flexWrap="wrap">
      {movies.map((movie) => (
        <Movie movie={movie} key={movie.id} />
      ))}
    </Flex>
  );
};
