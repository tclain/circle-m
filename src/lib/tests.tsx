import React from "react";
import { ThemeProvider } from "@chakra-ui/core";
import { render } from "@testing-library/react";

export const renderInTestMode = (fragment: React.ReactElement<any>) => {
  return render(<ThemeProvider>{fragment}</ThemeProvider>);
};
