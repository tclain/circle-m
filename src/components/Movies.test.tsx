import React from "react";
import { exampleApiResponse } from "../fixtures";
import { renderInTestMode } from "../lib/tests";
import { Movies } from "./Movies";

describe("Movies", () => {
  it("should render a list of movies", () => {
    expect(
      // FIXME: fix the types
      renderInTestMode(<Movies movies={exampleApiResponse.results as any} />)
    ).toMatchSnapshot();
  });
});
