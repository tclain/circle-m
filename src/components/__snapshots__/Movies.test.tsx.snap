// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movies should render a list of movies 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div
      class="Toaster"
      id="react-toast"
    >
      <span
        class="Toaster__manager-top"
        style="max-width: 560px; position: fixed; z-index: 5500; pointer-events: none; margin: 0px auto; text-align: center; top: 0px; right: 0px; left: 0px;"
      />
      <span
        class="Toaster__manager-top-left"
        style="max-width: 560px; position: fixed; z-index: 5500; pointer-events: none; top: 0px; left: 0px;"
      />
      <span
        class="Toaster__manager-top-right"
        style="max-width: 560px; position: fixed; z-index: 5500; pointer-events: none; top: 0px; right: 0px;"
      />
      <span
        class="Toaster__manager-bottom-left"
        style="max-width: 560px; position: fixed; z-index: 5500; pointer-events: none; bottom: 0px; left: 0px;"
      />
      <span
        class="Toaster__manager-bottom"
        style="max-width: 560px; position: fixed; z-index: 5500; pointer-events: none; margin: 0px auto; text-align: center; bottom: 0px; right: 0px; left: 0px;"
      />
      <span
        class="Toaster__manager-bottom-right"
        style="max-width: 560px; position: fixed; z-index: 5500; pointer-events: none; bottom: 0px; right: 0px;"
      />
    </div>
    <div>
      <div
        class="css-2imjyh"
      >
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/RYMX2wcKCBAr24UyPD7xwmjaTn.jpg"
          />
          <p
            class="css-11kknmz"
          >
            The Avengers
          </p>
          <p
            class="css-o75fak"
          >
            When an unexpected enemy emerges and threatens global safety and security, Nick Fury, director of the international peacekeeping agency known as S.H.I.E.L.D., finds himself in need of a team to pull the world back from the brink of disaster. Spanning the globe, a daring recruitment effort begins!
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/or06FN3Dka5tukK1e9sl16pB3iy.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Avengers: Endgame
          </p>
          <p
            class="css-o75fak"
          >
            After the devastating events of Avengers: Infinity War, the universe is in ruins due to the efforts of the Mad Titan, Thanos. With the help of remaining allies, the Avengers must assemble once more in order to undo Thanos' actions and restore order to the universe once and for all, no matter what consequences may be in store.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/7WsyChQLEftFiDOVTGkv3hFpyyt.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Avengers: Infinity War
          </p>
          <p
            class="css-o75fak"
          >
            As the Avengers and their allies have continued to protect the world from threats too large for any one hero to handle, a new danger has emerged from the cosmic shadows: Thanos. A despot of intergalactic infamy, his goal is to collect all six Infinity Stones, artifacts of unimaginable power, and use them to inflict his twisted will on all of reality. Everything the Avengers have fought for has led up to this moment - the fate of Earth and existence itself has never been more uncertain.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/4ssDuvEDkSArWEdyBl2X5EHvYKU.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Avengers: Age of Ultron
          </p>
          <p
            class="css-o75fak"
          >
            When Tony Stark tries to jumpstart a dormant peacekeeping program, things go awry and Earth’s Mightiest Heroes are put to the ultimate test as the fate of the planet hangs in the balance. As the villainous Ultron emerges, it is up to The Avengers to stop him from enacting his terrible plans, and soon uneasy alliances and unexpected action pave the way for an epic and unique global adventure.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/rXPxSAYf3FdQktnOtQnvhUPMhFp.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Avengement
          </p>
          <p
            class="css-o75fak"
          >
            While on a prison furlough, a lowly criminal evades his guards and returns to his old stomping ground to take revenge on the people who turned him into a cold blooded killer.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/1p5thyQ4pCy876HpdvFARqJ62N9.jpg"
          />
          <p
            class="css-11kknmz"
          >
            The Avengers
          </p>
          <p
            class="css-o75fak"
          >
            British Ministry agent John Steed, under direction from "Mother", investigates a diabolical plot by arch-villain Sir August de Wynter to rule the world with his weather control machine. Steed investigates the beautiful Doctor Mrs. Emma Peel, the only suspect, but simultaneously falls for her and joins forces with her to combat Sir August.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/1SbBKCbnULACOqWKN7eLfTu1gVm.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Avengers Grimm
          </p>
          <p
            class="css-o75fak"
          >
            When Rumpelstiltskin destroys the Magic Mirror and escapes to the modern world, the four princesses of "Once Upon a Time"-Cinderella, Sleeping Beauty, Snow White, and Rapunzel-are sucked through the portal too. Well-trained and endowed with magical powers, the four women must fight Rumpelstiltskin and his army of thralls before he enslaves everyone on Earth.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/iMCkGHVrYRdqKROPRPmVaJVSlg3.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Ultimate Avengers: The Movie
          </p>
          <p
            class="css-o75fak"
          >
            When a nuclear missile was fired at Washington in 1945, Captain America managed to detonate it in the upper atmosphere. But then he fell miles into the icy depths of the North Atlantic, where he remained lost for over sixty years. But now, with the world facing the very same evil, Captain America must rise again as our last hope for survival.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/4Au5QB55O5DWW0hXw1SQQDFIjAp.jpg"
          />
          <p
            class="css-11kknmz"
          >
            アベンジャーズ コンフィデンシャル：ブラック・ウィドウ ＆ パニッシャー
          </p>
          <p
            class="css-o75fak"
          >
            When the Punisher takes out a black-market weapons dealer, he stumbles upon a far-reaching terrorist plot devised by a group known as Leviathan.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/vSNxAJTlD0r02V9sPYpOjqDZXUK.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Captain America: The First Avenger
          </p>
          <p
            class="css-o75fak"
          >
            During World War II, Steve Rogers is a sickly man from Brooklyn who's transformed into super-soldier Captain America to aid in the war effort. Rogers must stop the Red Skull – Adolf Hitler's ruthless head of weaponry, and the leader of an organization that intends to use a mysterious device of untold powers for world domination.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/sMFyYZR9krqcQC99G6jnb10Zv4P.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Ultimate Avengers 2: Rise of the Panther
          </p>
          <p
            class="css-o75fak"
          >
            Mysterious Wakanda lies in the darkest heart of Africa, unknown to most of the world. An isolated land hidden behind closed borders, fiercely protected by its young king - the Black Panther. But when brutal alien invaders attack, the threat leaves the Black Panther with no option but to go against the sacred decrees of his people and ask for help from outsiders.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/kHQ81KIAq8kD9NfUmHjTgAahoCW.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Avenger
          </p>
          <p
            class="css-o75fak"
          >
            After his own daughter was killed in Panama in 1994, former CIA agent Calvin Dexter became a private 'specialist' in cases which wouldn't reach justice trough the regular legal channels. Two years later he accepts to find Richard 'Ricky' Edmunds for his pa, influential rich businessman Stephen Edmonds. Ricky for a private Canadian war victims charity in Bosnia and went missing. Dexter discovers Ricky was beaten to pulp and drowned for no other crime then helping street boys from the other side by Zoran Zilic and his Serbian paramilitary 'order'. He offers Steven to 'finish the job' as such war criminals don't go to trial. But deputy CIA director Paul Devereaux cares only for a nuclear arms project he wants to use Zilic for. So CIA troubleshooter Frank McBride is ordered to protect him and handle Dexter.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/qRbAr8yilTnXssmk52po7RyqZSH.jpg"
          />
          <p
            class="css-11kknmz"
          >
            The Toxic Avenger
          </p>
          <p
            class="css-o75fak"
          >
            Tromaville has a monstrous new hero. The Toxic Avenger is born when mop boy Melvin Junko falls into a vat of toxic waste. Now evildoers will have a lot to lose.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/fpG1NDbcLV2a7c8X7LC4FPISBT7.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Next Avengers: Heroes of Tomorrow
          </p>
          <p
            class="css-o75fak"
          >
            The children of the Avengers hone their powers and go head to head with the very enemy responsible for their parents' demise.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/jcoaZNyoN2l8G2RCTgcogaj525G.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Valentine
          </p>
          <p
            class="css-o75fak"
          >
            Batavia City, a beautiful city that no longer safely inhabited. Robbery, violence, and a variety of increasingly rampant criminality. In the midst of this chaotic city, Srimaya, a cafe waitress who dreams of becoming an actress never thought that dream would change her life. Her meeting with Bono, a film director and his friend, Wawan, will take her on a dangerous adventure full of thrilling action with lives at stake. Turning her from an ordinary girl into Batavia City heroine of hope, Valentine.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/evB9pQHF1qR2UQtuApbaigcZwG5.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Citizen Toxie: The Toxic Avenger IV
          </p>
          <p
            class="css-o75fak"
          >
            When the notorious Diaper Mafia take hostage the Tromaville School for the Very Special, only the Toxic Avenger and his morbidly obese sidekick Lardass can save Tromaville.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/hl4mI28D4NhE6ePNgPojQuyEYw7.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Avenging Angelo
          </p>
          <p
            class="css-o75fak"
          >
            A woman who has recently discovered that she is the daughter of Angelo, a major mafia boss, decides to wreak vengeance when he is killed by a hitman. She's aided by his faithful bodyguard, with whom she soon falls in love.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/uJ5pZZJW2bHo5tKFARTMpo7js3i.jpg"
          />
          <p
            class="css-11kknmz"
          >
            The Toxic Avenger Part II
          </p>
          <p
            class="css-o75fak"
          >
            The Toxic Avenger is lured to Tokyo, Japan by the evil corporation Apocalypse Inc. So while the Toxic Avenger is fighting crime in Tokyo, Apocalypse Inc. spread evil in Tromaville.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/xUBZNoY7idPfqKZepnDEv7Qc8GC.jpg"
          />
          <p
            class="css-11kknmz"
          >
            LEGO Marvel Super Heroes: Avengers Reassembled!
          </p>
          <p
            class="css-o75fak"
          >
            In "LEGO Marvel Super Heroes: Avengers Reassembled!," the Avengers prepare for a party at Stark Tower and notice that Iron Man is acting strange. After some investigation, they discover that the evil Ultron has taken control of Iron Man and his armor as part of his scheme to take over the world. It's up to Captain America, the Hulk, Thor, Vision, Black Widow, Hawkeye and their friends (Spider-Man, Iron Spider, and special guest star Ant-Man) to rescue Iron Man from Ultron's clutches and defeat the super villain before he causes even more destruction and chaos.
          </p>
        </div>
        <div
          class="css-ddntvx"
          data-testid="movie"
        >
          <img
            src="https://image.tmdb.org/t/p/w300/xfAcu74DRQXeM9XqFcE5MrSRzYP.jpg"
          />
          <p
            class="css-11kknmz"
          >
            Avengers Grimm: Time Wars
          </p>
          <p
            class="css-o75fak"
          >
            When Rumpelstiltskin tries to take over Earth once and for all, The Avengers Grimm must track him down through time in order to defeat him.
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-2imjyh"
    >
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/RYMX2wcKCBAr24UyPD7xwmjaTn.jpg"
        />
        <p
          class="css-11kknmz"
        >
          The Avengers
        </p>
        <p
          class="css-o75fak"
        >
          When an unexpected enemy emerges and threatens global safety and security, Nick Fury, director of the international peacekeeping agency known as S.H.I.E.L.D., finds himself in need of a team to pull the world back from the brink of disaster. Spanning the globe, a daring recruitment effort begins!
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/or06FN3Dka5tukK1e9sl16pB3iy.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Avengers: Endgame
        </p>
        <p
          class="css-o75fak"
        >
          After the devastating events of Avengers: Infinity War, the universe is in ruins due to the efforts of the Mad Titan, Thanos. With the help of remaining allies, the Avengers must assemble once more in order to undo Thanos' actions and restore order to the universe once and for all, no matter what consequences may be in store.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/7WsyChQLEftFiDOVTGkv3hFpyyt.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Avengers: Infinity War
        </p>
        <p
          class="css-o75fak"
        >
          As the Avengers and their allies have continued to protect the world from threats too large for any one hero to handle, a new danger has emerged from the cosmic shadows: Thanos. A despot of intergalactic infamy, his goal is to collect all six Infinity Stones, artifacts of unimaginable power, and use them to inflict his twisted will on all of reality. Everything the Avengers have fought for has led up to this moment - the fate of Earth and existence itself has never been more uncertain.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/4ssDuvEDkSArWEdyBl2X5EHvYKU.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Avengers: Age of Ultron
        </p>
        <p
          class="css-o75fak"
        >
          When Tony Stark tries to jumpstart a dormant peacekeeping program, things go awry and Earth’s Mightiest Heroes are put to the ultimate test as the fate of the planet hangs in the balance. As the villainous Ultron emerges, it is up to The Avengers to stop him from enacting his terrible plans, and soon uneasy alliances and unexpected action pave the way for an epic and unique global adventure.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/rXPxSAYf3FdQktnOtQnvhUPMhFp.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Avengement
        </p>
        <p
          class="css-o75fak"
        >
          While on a prison furlough, a lowly criminal evades his guards and returns to his old stomping ground to take revenge on the people who turned him into a cold blooded killer.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/1p5thyQ4pCy876HpdvFARqJ62N9.jpg"
        />
        <p
          class="css-11kknmz"
        >
          The Avengers
        </p>
        <p
          class="css-o75fak"
        >
          British Ministry agent John Steed, under direction from "Mother", investigates a diabolical plot by arch-villain Sir August de Wynter to rule the world with his weather control machine. Steed investigates the beautiful Doctor Mrs. Emma Peel, the only suspect, but simultaneously falls for her and joins forces with her to combat Sir August.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/1SbBKCbnULACOqWKN7eLfTu1gVm.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Avengers Grimm
        </p>
        <p
          class="css-o75fak"
        >
          When Rumpelstiltskin destroys the Magic Mirror and escapes to the modern world, the four princesses of "Once Upon a Time"-Cinderella, Sleeping Beauty, Snow White, and Rapunzel-are sucked through the portal too. Well-trained and endowed with magical powers, the four women must fight Rumpelstiltskin and his army of thralls before he enslaves everyone on Earth.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/iMCkGHVrYRdqKROPRPmVaJVSlg3.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Ultimate Avengers: The Movie
        </p>
        <p
          class="css-o75fak"
        >
          When a nuclear missile was fired at Washington in 1945, Captain America managed to detonate it in the upper atmosphere. But then he fell miles into the icy depths of the North Atlantic, where he remained lost for over sixty years. But now, with the world facing the very same evil, Captain America must rise again as our last hope for survival.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/4Au5QB55O5DWW0hXw1SQQDFIjAp.jpg"
        />
        <p
          class="css-11kknmz"
        >
          アベンジャーズ コンフィデンシャル：ブラック・ウィドウ ＆ パニッシャー
        </p>
        <p
          class="css-o75fak"
        >
          When the Punisher takes out a black-market weapons dealer, he stumbles upon a far-reaching terrorist plot devised by a group known as Leviathan.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/vSNxAJTlD0r02V9sPYpOjqDZXUK.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Captain America: The First Avenger
        </p>
        <p
          class="css-o75fak"
        >
          During World War II, Steve Rogers is a sickly man from Brooklyn who's transformed into super-soldier Captain America to aid in the war effort. Rogers must stop the Red Skull – Adolf Hitler's ruthless head of weaponry, and the leader of an organization that intends to use a mysterious device of untold powers for world domination.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/sMFyYZR9krqcQC99G6jnb10Zv4P.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Ultimate Avengers 2: Rise of the Panther
        </p>
        <p
          class="css-o75fak"
        >
          Mysterious Wakanda lies in the darkest heart of Africa, unknown to most of the world. An isolated land hidden behind closed borders, fiercely protected by its young king - the Black Panther. But when brutal alien invaders attack, the threat leaves the Black Panther with no option but to go against the sacred decrees of his people and ask for help from outsiders.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/kHQ81KIAq8kD9NfUmHjTgAahoCW.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Avenger
        </p>
        <p
          class="css-o75fak"
        >
          After his own daughter was killed in Panama in 1994, former CIA agent Calvin Dexter became a private 'specialist' in cases which wouldn't reach justice trough the regular legal channels. Two years later he accepts to find Richard 'Ricky' Edmunds for his pa, influential rich businessman Stephen Edmonds. Ricky for a private Canadian war victims charity in Bosnia and went missing. Dexter discovers Ricky was beaten to pulp and drowned for no other crime then helping street boys from the other side by Zoran Zilic and his Serbian paramilitary 'order'. He offers Steven to 'finish the job' as such war criminals don't go to trial. But deputy CIA director Paul Devereaux cares only for a nuclear arms project he wants to use Zilic for. So CIA troubleshooter Frank McBride is ordered to protect him and handle Dexter.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/qRbAr8yilTnXssmk52po7RyqZSH.jpg"
        />
        <p
          class="css-11kknmz"
        >
          The Toxic Avenger
        </p>
        <p
          class="css-o75fak"
        >
          Tromaville has a monstrous new hero. The Toxic Avenger is born when mop boy Melvin Junko falls into a vat of toxic waste. Now evildoers will have a lot to lose.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/fpG1NDbcLV2a7c8X7LC4FPISBT7.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Next Avengers: Heroes of Tomorrow
        </p>
        <p
          class="css-o75fak"
        >
          The children of the Avengers hone their powers and go head to head with the very enemy responsible for their parents' demise.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/jcoaZNyoN2l8G2RCTgcogaj525G.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Valentine
        </p>
        <p
          class="css-o75fak"
        >
          Batavia City, a beautiful city that no longer safely inhabited. Robbery, violence, and a variety of increasingly rampant criminality. In the midst of this chaotic city, Srimaya, a cafe waitress who dreams of becoming an actress never thought that dream would change her life. Her meeting with Bono, a film director and his friend, Wawan, will take her on a dangerous adventure full of thrilling action with lives at stake. Turning her from an ordinary girl into Batavia City heroine of hope, Valentine.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/evB9pQHF1qR2UQtuApbaigcZwG5.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Citizen Toxie: The Toxic Avenger IV
        </p>
        <p
          class="css-o75fak"
        >
          When the notorious Diaper Mafia take hostage the Tromaville School for the Very Special, only the Toxic Avenger and his morbidly obese sidekick Lardass can save Tromaville.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/hl4mI28D4NhE6ePNgPojQuyEYw7.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Avenging Angelo
        </p>
        <p
          class="css-o75fak"
        >
          A woman who has recently discovered that she is the daughter of Angelo, a major mafia boss, decides to wreak vengeance when he is killed by a hitman. She's aided by his faithful bodyguard, with whom she soon falls in love.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/uJ5pZZJW2bHo5tKFARTMpo7js3i.jpg"
        />
        <p
          class="css-11kknmz"
        >
          The Toxic Avenger Part II
        </p>
        <p
          class="css-o75fak"
        >
          The Toxic Avenger is lured to Tokyo, Japan by the evil corporation Apocalypse Inc. So while the Toxic Avenger is fighting crime in Tokyo, Apocalypse Inc. spread evil in Tromaville.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/xUBZNoY7idPfqKZepnDEv7Qc8GC.jpg"
        />
        <p
          class="css-11kknmz"
        >
          LEGO Marvel Super Heroes: Avengers Reassembled!
        </p>
        <p
          class="css-o75fak"
        >
          In "LEGO Marvel Super Heroes: Avengers Reassembled!," the Avengers prepare for a party at Stark Tower and notice that Iron Man is acting strange. After some investigation, they discover that the evil Ultron has taken control of Iron Man and his armor as part of his scheme to take over the world. It's up to Captain America, the Hulk, Thor, Vision, Black Widow, Hawkeye and their friends (Spider-Man, Iron Spider, and special guest star Ant-Man) to rescue Iron Man from Ultron's clutches and defeat the super villain before he causes even more destruction and chaos.
        </p>
      </div>
      <div
        class="css-ddntvx"
        data-testid="movie"
      >
        <img
          src="https://image.tmdb.org/t/p/w300/xfAcu74DRQXeM9XqFcE5MrSRzYP.jpg"
        />
        <p
          class="css-11kknmz"
        >
          Avengers Grimm: Time Wars
        </p>
        <p
          class="css-o75fak"
        >
          When Rumpelstiltskin tries to take over Earth once and for all, The Avengers Grimm must track him down through time in order to defeat him.
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
